src/dfcx_scrapi/core/conversation.py

```diff
@@ -334,20 +393,21 @@ def _get_intent_detection(self, test_set: pd.DataFrame):
             on=["flow_display_name", "page_display_name"],
             how="left",
         )
-        utterances = list(test_set_mapped["utterance"])
         page_ids = list(test_set_mapped["page_id"])
+        send_objs = self._extract_send_objs(test_set_mapped)
```

Comment:
let's remove changes to `_get_intent_detection` for this PR

Reply:
reverted